v2.0/foreign-key.md

```diff
@@ -119,25 +131,48 @@ CREATE TABLE packages (
   ;
 ~~~
 
-## Usage Example
+## Usage Examples
+
+### Use a Foreign Key Constraint with Default Actions
+
+In this example, we'll create a table with a foreign key constraint with the default [actions](#foreign-key-actions) (`ON UPDATE NO ACTION` / `ON DELETE NO ACTION`).
 
+First, create the referenced table.
```

Comment:
nit: Whenever you are introducing a command, I'd suggest using a colon (this should eventually be in our style guide).

> First, create the referenced table:

Reply:
Done